constructs/Compute/virtualMachinesMultiple/main.bicep

```diff
@@ -285,7 +285,7 @@ param timeZone string = ''
 param additionalUnattendContent array = []
 
 @description('Optional. Specifies the Windows Remote Management listeners. This enables remote Windows PowerShell. - WinRMConfiguration object.')
```

Comment:
I do want to mention that this looks like it meant to be a type [WinRMConfiguration](https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/create-or-update?view=rest-compute-2023-07-01) but in actuality its [WinRMListener](https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/create-or-update?view=rest-compute-2023-07-01). Maybe the solution is to remove the listener mapping and allow users to pass the WinRMConfiguration type?